pom.xml

```diff
@@ -179,7 +179,7 @@
     <antlr4.version>4.5.3</antlr4.version>
     <jpam.version>1.1</jpam.version>
     <selenium.version>2.52.0</selenium.version>
-    <paranamer.version>2.8</paranamer.version>
+    <paranamer.version>2.6</paranamer.version>
```

Comment:
Yep, looks like this was added by mistake: https://github.com/yhuai/spark/commit/fa4bc8ea8bab1277d1482da370dac79947cac719#diff-600376dffeb79835ede4a0b285078036R183